[debug] [2024-09-14T22:26:27.637Z] ----------------------------------------------------------------------
[debug] [2024-09-14T22:26:27.639Z] Command:       /home/dev/.nvm/versions/node/v22.6.0/bin/node /home/dev/.nvm/versions/node/v22.6.0/bin/firebase deploy
[debug] [2024-09-14T22:26:27.640Z] CLI Version:   13.17.0
[debug] [2024-09-14T22:26:27.640Z] Platform:      linux
[debug] [2024-09-14T22:26:27.640Z] Node Version:  v22.6.0
[debug] [2024-09-14T22:26:27.640Z] Time:          Sat Sep 14 2024 22:26:27 GMT+0000 (Greenwich Mean Time)
[debug] [2024-09-14T22:26:27.641Z] ----------------------------------------------------------------------
[debug] 
[debug] [2024-09-14T22:26:32.718Z] Error: Timed out.
    at Timeout._onTimeout (/home/dev/.nvm/versions/node/v22.6.0/lib/node_modules/firebase-tools/lib/utils.js:268:49)
    at listOnTimeout (node:internal/timers:581:17)
    at process.processTimers (node:internal/timers:519:7)
[error] 
[error] Error: An unexpected error has occurred.
[debug] [2024-09-14T22:27:16.589Z] ----------------------------------------------------------------------
[debug] [2024-09-14T22:27:16.592Z] Command:       /home/dev/.nvm/versions/node/v22.6.0/bin/node /home/dev/.nvm/versions/node/v22.6.0/bin/firebase init
[debug] [2024-09-14T22:27:16.592Z] CLI Version:   13.17.0
[debug] [2024-09-14T22:27:16.592Z] Platform:      linux
[debug] [2024-09-14T22:27:16.592Z] Node Version:  v22.6.0
[debug] [2024-09-14T22:27:16.593Z] Time:          Sat Sep 14 2024 22:27:16 GMT+0000 (Greenwich Mean Time)
[debug] [2024-09-14T22:27:16.593Z] ----------------------------------------------------------------------
[debug] 
[debug] [2024-09-14T22:27:16.601Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2024-09-14T22:27:16.602Z] > authorizing via signed-in user (devfronts45@gmail.com)
[info] 
     ######## #### ########  ######## ########     ###     ######  ########
     ##        ##  ##     ## ##       ##     ##  ##   ##  ##       ##
     ######    ##  ########  ######   ########  #########  ######  ######
     ##        ##  ##    ##  ##       ##     ## ##     ##       ## ##
     ##       #### ##     ## ######## ########  ##     ##  ######  ########

You're about to initialize a Firebase project in this directory:

  /home/dev/Documents/Projets/Current/Web/nestor_store01

[debug] [2024-09-14T22:27:24.452Z] Error: Timed out.
    at Timeout._onTimeout (/home/dev/.nvm/versions/node/v22.6.0/lib/node_modules/firebase-tools/lib/utils.js:268:49)
    at listOnTimeout (node:internal/timers:581:17)
    at process.processTimers (node:internal/timers:519:7)
[error] 
[error] Error: An unexpected error has occurred.
[debug] [2024-09-14T22:27:32.124Z] ----------------------------------------------------------------------
[debug] [2024-09-14T22:27:32.126Z] Command:       /home/dev/.nvm/versions/node/v22.6.0/bin/node /home/dev/.nvm/versions/node/v22.6.0/bin/firebase init
[debug] [2024-09-14T22:27:32.127Z] CLI Version:   13.17.0
[debug] [2024-09-14T22:27:32.127Z] Platform:      linux
[debug] [2024-09-14T22:27:32.127Z] Node Version:  v22.6.0
[debug] [2024-09-14T22:27:32.127Z] Time:          Sat Sep 14 2024 22:27:32 GMT+0000 (Greenwich Mean Time)
[debug] [2024-09-14T22:27:32.127Z] ----------------------------------------------------------------------
[debug] 
[debug] [2024-09-14T22:27:32.135Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2024-09-14T22:27:32.135Z] > authorizing via signed-in user (devfronts45@gmail.com)
[info] 
     ######## #### ########  ######## ########     ###     ######  ########
     ##        ##  ##     ## ##       ##     ##  ##   ##  ##       ##
     ######    ##  ########  ######   ########  #########  ######  ######
     ##        ##  ##    ##  ##       ##     ## ##     ##       ## ##
     ##       #### ##     ## ######## ########  ##     ##  ######  ########

You're about to initialize a Firebase project in this directory:

  /home/dev/Documents/Projets/Current/Web/nestor_store01

[debug] [2024-09-14T22:27:43.097Z] Error: Timed out.
    at Timeout._onTimeout (/home/dev/.nvm/versions/node/v22.6.0/lib/node_modules/firebase-tools/lib/utils.js:268:49)
    at listOnTimeout (node:internal/timers:581:17)
    at process.processTimers (node:internal/timers:519:7)
[error] 
[error] Error: An unexpected error has occurred.
[debug] [2024-09-14T22:27:52.496Z] ----------------------------------------------------------------------
[debug] [2024-09-14T22:27:52.498Z] Command:       /home/dev/.nvm/versions/node/v22.6.0/bin/node /home/dev/.nvm/versions/node/v22.6.0/bin/firebase deploy
[debug] [2024-09-14T22:27:52.499Z] CLI Version:   13.17.0
[debug] [2024-09-14T22:27:52.499Z] Platform:      linux
[debug] [2024-09-14T22:27:52.499Z] Node Version:  v22.6.0
[debug] [2024-09-14T22:27:52.499Z] Time:          Sat Sep 14 2024 22:27:52 GMT+0000 (Greenwich Mean Time)
[debug] [2024-09-14T22:27:52.499Z] ----------------------------------------------------------------------
[debug] 
[debug] [2024-09-14T22:27:57.564Z] Error: Timed out.
    at Timeout._onTimeout (/home/dev/.nvm/versions/node/v22.6.0/lib/node_modules/firebase-tools/lib/utils.js:268:49)
    at listOnTimeout (node:internal/timers:581:17)
    at process.processTimers (node:internal/timers:519:7)
[error] 
[error] Error: An unexpected error has occurred.
[debug] [2024-09-14T22:29:59.127Z] ----------------------------------------------------------------------
[debug] [2024-09-14T22:29:59.132Z] Command:       /home/dev/.nvm/versions/node/v22.6.0/bin/node /home/dev/.nvm/versions/node/v22.6.0/bin/firebase deploy
[debug] [2024-09-14T22:29:59.133Z] CLI Version:   13.17.0
[debug] [2024-09-14T22:29:59.134Z] Platform:      linux
[debug] [2024-09-14T22:29:59.134Z] Node Version:  v22.6.0
[debug] [2024-09-14T22:29:59.135Z] Time:          Sat Sep 14 2024 22:29:59 GMT+0000 (Greenwich Mean Time)
[debug] [2024-09-14T22:29:59.135Z] ----------------------------------------------------------------------
[debug] 
[debug] [2024-09-14T22:30:04.373Z] Error: Timed out.
    at Timeout._onTimeout (/home/dev/.nvm/versions/node/v22.6.0/lib/node_modules/firebase-tools/lib/utils.js:268:49)
    at listOnTimeout (node:internal/timers:581:17)
    at process.processTimers (node:internal/timers:519:7)
[error] 
[error] Error: An unexpected error has occurred.
[debug] [2024-09-15T18:57:38.897Z] ----------------------------------------------------------------------
[debug] [2024-09-15T18:57:38.900Z] Command:       /home/dev/.nvm/versions/node/v22.6.0/bin/node /home/dev/.nvm/versions/node/v22.6.0/bin/firebase init
[debug] [2024-09-15T18:57:38.900Z] CLI Version:   13.17.0
[debug] [2024-09-15T18:57:38.900Z] Platform:      linux
[debug] [2024-09-15T18:57:38.900Z] Node Version:  v22.6.0
[debug] [2024-09-15T18:57:38.901Z] Time:          Sun Sep 15 2024 18:57:38 GMT+0000 (Greenwich Mean Time)
[debug] [2024-09-15T18:57:38.901Z] ----------------------------------------------------------------------
[debug] 
[debug] [2024-09-15T18:57:38.911Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2024-09-15T18:57:38.912Z] > authorizing via signed-in user (devfronts45@gmail.com)
[info] 
     ######## #### ########  ######## ########     ###     ######  ########
     ##        ##  ##     ## ##       ##     ##  ##   ##  ##       ##
     ######    ##  ########  ######   ########  #########  ######  ######
     ##        ##  ##    ##  ##       ##     ## ##     ##       ## ##
     ##       #### ##     ## ######## ########  ##     ##  ######  ########

You're about to initialize a Firebase project in this directory:

  /home/dev/Documents/Projets/Current/Web/nestor_store01

[debug] [2024-09-15T18:58:21.550Z] Error: Timed out.
    at Timeout._onTimeout (/home/dev/.nvm/versions/node/v22.6.0/lib/node_modules/firebase-tools/lib/utils.js:268:49)
    at listOnTimeout (node:internal/timers:581:17)
    at process.processTimers (node:internal/timers:519:7)
[error] 
[error] Error: An unexpected error has occurred.
